azurerm/internal/services/apimanagement/api_management_subscription_resource.go

```diff
@@ -187,13 +187,19 @@ func resourceArmApiManagementSubscriptionRead(d *schema.ResourceData, meta inter
 
 	if props := resp.SubscriptionContractProperties; props != nil {
 		d.Set("display_name", props.DisplayName)
-		d.Set("primary_key", props.PrimaryKey)
-		d.Set("secondary_key", props.SecondaryKey)
 		d.Set("state", string(props.State))
 		d.Set("product_id", props.Scope)
 		d.Set("user_id", props.OwnerID)
 	}
 
+	// Primary and secondary keys must be got from this additional api
```

Comment:
Was this documented as a breaking change from Microsoft? Just curious how you figured this out.

Reply:
The service team comment it in swagger